src/cyclotom.c

```diff
@@ -985,6 +986,9 @@ Obj             AInvCyc (
             exs = EXPOS_CYC(op,len);
             exp = EXPOS_CYC(res,len);
         }
+        else {
+            prd = AInvInt(cfs[i]);
+        }
```

Comment:
Also, I just read the comment at the start of this file, which states that the coefficients can be rationals, so `AInvInt` is not correct either way (I know I suggested it earlier, but that was without reading the code)